survey-runner-vpc/subnets.tf

```diff
@@ -13,11 +13,12 @@ resource "aws_subnet" "database" {
 }
 
 resource "aws_db_subnet_group" "eq_rds" {
-  name        = "${var.env}-eq-rds"
+  name        = "${var.env}-eq-${var.db_subnet_group_identifier}rds"
```

Comment:
@ajmaddaford adding this optional identifier allows me to make the author name unique without needing to change the Runner one.
In theory the Runner DB would go soon this shouldn't be suck a big issue?